index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section></section>
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo">John doe </div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+     </nav>
```

Comment:
- [ ] Well done on creating the nav element, however, the requirement asks to wrap it inside of a header tag instead, Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation, please fix this.
- [ ] The logo name is typically used as a link to the home page, this is why it is recommended to use an `a` tag instead of a div, please update this issue.
- [ ] I think you can change the logo name from john doe to your correct name or initials

`HINT` : 
```
<header>
    <a> logo name </a>
    <nav>
       - item
       - item
     </nav>
</header>

```